patches/enable-state-partitioning.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/net/base/features.cc b/net/base/features.cc
+index 387c7f01e1f5e..a69ddbc33983f 100644
+--- a/net/base/features.cc
++++ b/net/base/features.cc
+@@ -134,7 +134,7 @@ BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
+ 
+ BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
+              "PartitionConnectionsByNetworkIsolationKey",
```

Comment:
This is already in https://github.com/secureblue/hardened-chromium/blob/master/vanadium_patches/0071-enable-partitioning-connections-by-default.patch